libs/payments/paypal/src/lib/paypalCustomer/paypalCustomer.factories.ts

```diff
@@ -0,0 +1,53 @@
+import { faker } from '@faker-js/faker';
+import {
+  CreatePaypalCustomer,
+  ResultPaypalCustomer,
+  UpdatePaypalCustomer,
+} from './paypalCustomer.types';
+
+export const ResultPaypalCustomerFactory = (
+  override?: Partial<ResultPaypalCustomer>
+): ResultPaypalCustomer => ({
+  ...override,
+  uid: faker.string.hexadecimal({
+    length: 32,
+    prefix: '',
+    casing: 'lower',
+  }),
+  billingAgreementId: faker.string.hexadecimal({
+    length: 10,
+    prefix: '',
+  }),
```

Comment:
Same Q here re: helper (we seemingly create `billingAgreementId` UUIDs 4&times; in this PR).

Reply:
See my reply to the other comment.